src/hooks/useOperationModal/Modal/BorrowForm/index.tsx

```diff
@@ -164,6 +161,15 @@ export const BorrowFormUi: React.FC<BorrowFormUiProps> = ({
         )}
       </div>
 
+      <LabeledInlineContent
+        label={t('operationModal.borrow.borrowableAmount')}
+        css={sharedStyles.getRow({ isLast: true })}
```

Comment:
Could be done in pure css with last child selector

Reply:
Same as previously